pkg/doc/verifiable/credential.go

```diff
@@ -1062,9 +1124,25 @@ func contextToSerialize(context []string, cContext []interface{}) interface{} {
 	return context
 }
 
+func typedIdsToSerialize(typedIds []TypedID) ([]byte, error) {
```

Comment:
ID should be capitalized

Reply:
(also the variables - typedIDs)

Reply:
Do you mean "ToSerialize**d**" instead of "ToSerialize"? As in a conversion of []TypedID into []byte?

Reply:
Naming is not good, I will think of some better names here...

Reply:
ok :)

Reply:
It's now `typedIDsToRaw()`. I also renamed functions similar to this one.